src/core/config.js

```diff
@@ -79,6 +79,16 @@ if ( urlParams.filter === true ) {
 // String search anywhere in moduleName+testName
 config.filter = urlParams.filter;
 
+// Randomize the order in which tests are run
+if ( urlParams.random ) {
+	config.random = parseInt( urlParams.random, 10 );
+
+	// Generate a random seed if the random option is specified with a non-number
+	if ( isNaN( config.random ) ) {
+		config.random = Math.floor( Math.random() * 10000 );
+	}
+}
```

Comment:
Taking the other comments into consideration, this would become something like:
```js
if ( urlParams.random ) {
	config.random = urlParams.random;
	if ( config.random === true ) {
		config.random = Math.random();
	}
}
```

So `?random` would yield a different seed each time, but e.g. `?random=foo` would always yield the same seed (and `?random=bar` would always yield its own unique seed).